webapi/src/test/scala/org/knora/webapi/slice/resourceinfo/repo/ResourceInfoRepoFake.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.resourceinfo.repo
+
+import zio.Ref
+import zio.Task
+import zio.UIO
+import zio.ULayer
+import zio.URIO
+import zio.ZIO
+import zio.ZLayer
+
+import org.knora.webapi.slice.resourceinfo.domain
+import org.knora.webapi.slice.resourceinfo.domain.InternalIri
+import org.knora.webapi.slice.resourceinfo.domain.ResourceInfo
+import org.knora.webapi.slice.resourceinfo.domain.ResourceInfoRepo
+
+final case class ResourceInfoRepoFake(entitiesRef: Ref[Map[(InternalIri, InternalIri), List[ResourceInfo]]])
```

Comment:
What do you mean with `Fake`? Shouldn't it be called `Mock` oder `Test`? What is the difference?

Reply:
This a nice consice article on the different types of test doubles: https://jesusvalerareales.medium.com/testing-with-test-doubles-7c3abb9eb3f2

Reply:
Thanks, that's interesting. If I understand it correctly, then what we called `Test` so far would be `Fake` according to this article. I think we should decide on either one and use that consistently. Otherwise it gets confusing over time. But maybe I don't see all the details and our `Test` and `Fake` implementations are different things after all.

Reply:
I think of using `Test` as an indicator to it being a _test double_ of whatever kind. In future we can think about being more precise in the naming. For example the article is very precise of what `Mock` should do. Whilst I like this understanding this only one possible definition for mocks. You will often find the term mock being used as a substitute to the more general _test double_.

Another more in depth article about test doubles and their uses can be found here:
https://martinfowler.com/articles/mocksArentStubs.html